spec/requests/api/v2/providers/accredited_body_filter_providers_spec.rb

```diff
@@ -51,6 +60,21 @@ def perform_request
       get request_path, headers: { "HTTP_AUTHORIZATION" => credentials }
     end
 
+    context "when accredited_provider has a course" do
```

Comment:
Similar to the comment about the `it "is returned"`, could we just be a little more clear here:

```suggestion
    context "when accredited_provider has a self-accredited course" do
```